### PR TITLE
solves #986

### PR DESCRIPTION
**Pull request by pakhandibaba**
@trang @jiru

### CODE REVIEW

**Comment by trang** [comment]
Thanks for your pull request, @pakhandibaba. I haven't reviewed it yet, but I'll have the same comments than the ones I wrote to achaitanyasai (cf. #1020).

* You don't need to ping anyone, we receive notifications for all pull requests.
* You can improve your future commit messages to contain more information about what you did.
* If you will be submitting more pull requests for us in the future, I recently wrote [guidelines][1]. Make sure to read them.

That being said, I hope I'll have time to review your pull request this weekend.

[1]:https://github.com/Tatoeba/tatoeba2/wiki/Guidelines-for-submitting-pull-requests

File: app/views/sentences/show_all_in.ctp [hunk 1]
```diff
@@ -27,18 +27,28 @@
 
 $languageName = $languages->codeToNameToFormat($lang);
 
-$title = format(__('All sentences in {language}', true),
-                array('language' => $languageName));
+if ( $filterAudioOnly == 'Yes' ) {
```
Comment (jiru) [inline app/views/sentences/show_all_in.ctp]: `$filterAudioOnly` can’t possibly have that value.

File: app/views/sentences/show_all_in.ctp [hunk 2]
```diff
@@ -27,18 +27,28 @@
...
     if ($notTranslatedInto == 'und') {
         $notTranslatedIntoName = __('any language', true);
     } else {
         $notTranslatedIntoName = $languages->codeToNameToFormat($notTranslatedInto);
     }
-    $title = format(
-        __('Sentences in {language} not translated into {translationLanguage}', true),
-        array('language' => $languageName, 'translationLanguage' => $notTranslatedIntoName)
-    );
+    if ( $filterAudioOnly == 'Yes' ) {
```
Comment (jiru) [inline app/views/sentences/show_all_in.ctp]: Same comment here. Also, please try to follow the code style of the existing code when adding yours:
```php
if ( a ) { # not OK
if (a) { # OK
```

**Comment by jiru** [comment]
Thank you for your PR, but it doesn’t work, likely because of what I said in my first comment.

**Pull request closed without merging**